cmd/flags/force.go

```diff
@@ -0,0 +1,32 @@
+package flags
+
+import (
+	"bufio"
+	"errors"
+	"fmt"
+	"os"
+	"strings"
+)
+
+var ErrExpectYesOrNo = errors.New("invalid answer, expect \"yes\" or \"no\"")
+
+func DoYouConfirm() (bool, error) {
+	reader := bufio.NewReader(os.Stdin)
+	fmt.Print("Are you sure? (yes/no) ") //nolint:forbidigo
```

Comment:
Just a quick change: I think it might be slightly better as`(y/n)` since usually people expect just to be able to enter `y` for yes.

Reply:
ok !